docs/tutorials/01-create-simple-blueprint.md

```diff
@@ -344,7 +353,7 @@ landscaper-cli blueprints push \
 Blueprints are also just resources/artifacts of a Component Descriptor. Therefore, after the Blueprint got uploaded, its reference needs to be added to the Component Descriptor. This is necessary to make sure that all resources of an application are known and stored - and the Blueprint is just one of the resources of an application.
 In addition, Landscaper needs this information to resolve the location of the Blueprint resource.
 
-Note that the repository context as well as the Blueprint resource should be added to the Component Descriptor.
+Note that the Blueprint resource should be added to the Component Descriptor.
```

Comment:
I would leave that comment or tell that the repository context should be replaced with the repository context for the own registry

Reply:
Ok, I have reverted the change of this sentence.